docs/extras/nandrestore.md

```diff
@@ -1,36 +1,39 @@
 	
-	-If you're going to restore an old nand which will downgrade your firmware it's best to create a second NAND backup before restoring the first one in case something goes wrong.
+	-If you're going to restore an old NAND which will downgrade your firmware it's best to create a second NAND backup before restoring the first one in case something goes wrong.
 
 ### What you need:
-- Your `Rawnand.bin` (Combined or in 15 or 30 parts)
+- Your `RawNAND.bin` (Combined or in 15 or 30 parts)
 - Your `BOOT0` and `BOOT1`
-	- If you're missing any of the files above, ask for further help in the [Nintendo Homebrew](https://discord.gg/C29hYvh) Discord server.
-- The [Hekate](https://github.com/CTCaer/hekate/releases/) Payload
+	- If you're missing any of the files above, ask for further help in the [Nintendo Homebrew](https://discord.gg/C29hYvh) Discord server
+- The <a href="https://github.com/CTCaer/hekate/releases/" target="_blank">Hekate</a> Payload
 - A microSD card larger than 32gb
 
 ### Instructions:
 
-Before we start, check if you have a tree of folders called `backup/[8 Character NAND id]/restore` on your sd card
+Before we start, check if you have a tree of folders called `backup/[8 Character NAND id]/restore` on your sd card.
 
-!!! warning "If you don't see a backup or [8 Character NAND id] folder on your sd card:"
-	1. Launch the latest hekate payload on your switch.
-	2. Navigate to `Tools > Backup > Backup eMMC BOOT0/1`, and press the power button to select the option.
-	3. Wait for the process to complete. When it's completed, you now have a `backup/[8 Character NAND id]/restore` folder on your sd card. Continue with step 1 of the guide.
+!!! Warning "If you don't see a backup or [8 Character NAND id] folder on your sd card:"
+	This means you do not have a nand backup, it is highly reccomended you make one as soon as possible. Follow the steps below to make one.
 
-1. Copy your `Rawnand.bin` (combined or in 15 or 30 parts), `BOOT0`, and `BOOT1` to the `backup/[8 Character NAND id]/restore` folder on the sd card
-2. Put your sd card into your switch, and launch hekate.
-3. Navigate to `Tools > Restore`. Select `Restore eMMC BOOT0/1`. Wait for this process to complete. After it finished, you can press power to return.
-4. Select `Restore eMMC RAW GPP`. This WILL take a very long time to complete. After it finished, you can press power to return.
+	1. Launch the latest Hekate payload on your switch.
+	2. Navigate to `Tools > Backup eMMC > eMMC BOOT0 & BOOT1` and let it do its thing.
+	3. Then go to `Tools > Backup eMMC > eMMC RAW GPP` and let it do its thing. (This will take a while!)
+	4. Wait for the process to complete. When it's completed, you now have a `backup/[8 Character NAND id]/restore` folder on your sd card. Continue with step 1 of the guide.
```

Comment:
Fairly certain this puts the instructions in the warnbox, not sure if this is intentional.

Reply:
It does, this was intentional. it was the original guide according to meme.

Reply:
https://nh-server.github.io/switch-guide/extras/nandrestore/ it's intentional

Reply:
Consider this irrelevant then.